### PR TITLE
tvOS support

### PR DESCRIPTION
**Pull request by ErikEverson**
Waiting on release with tvOS support before merging. 

- Added tvOS support by flagging out features that are not available for tvOS.
- Fixed layout issues on tvOS and iPad

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -30,7 +31,7 @@ let package = Package(
     ],
     dependencies: [
         // Ditto.diskUsage was added in 3.0.1
-        .package(url: "https://github.com/getditto/DittoSwiftPackage", from: "4.0.0"),
+        .package(path: "../DittoSwiftPackage"),
```
Comment (bplattenburg) [inline Package.swift:34]: I think this needs to change back to the git URL rather than a local path

**Review by bplattenburg — CHANGES_REQUESTED** [review]

File: Package.swift (hunk 1 above)
Comment (bplattenburg) [reply Package.swift:34]: (Which I assume is because this is a draft)

**Comment by ErikEverson** [comment]
Here are screen shots of before 
Simulator Screenshot - Apple TV 4K - 2024-05-24 at 09 13 55
Simulator Screenshot - Apple TV 4K - 2024-05-24 at 09 14 13
Simulator Screenshot - Apple TV 4K - 2024-05-24 at 09 14 39
Simulator Screenshot - Apple TV 4K - 2024-05-24 at 09 15 16
Simulator Screenshot - Apple TV 4K - 2024-05-24 at 09 15 36
Simulator Screenshot - Apple TV 4K - 2024-05-24 at 09 15 58
Simulator Screenshot - Apple TV 4K - 2024-05-24 at 09 16 28
Simulator Screenshot - Apple TV 4K - 2024-05-24 at 09 16 41
Simulator Screenshot - Apple TV 4K - 2024-05-24 at 09 17 04
Simulator Screenshot - iPad (10th generation) - 2024-05-24 at 09 13 57
Simulator Screenshot - iPad (10th generation) - 2024-05-24 at 09 14 14
Simulator Screenshot - iPad (10th generation) - 2024-05-24 at 09 14 40
Simulator Screenshot - iPad (10th generation) - 2024-05-24 at 09 15 18
Simulator Screenshot - iPad (10th generation) - 2024-05-24 at 09 15 37
Simulator Screenshot - iPad (10th generation) - 2024-05-24 at 09 15 59
Simulator Screenshot - iPad (10th generation) - 2024-05-24 at 09 16 29
Simulator Screenshot - iPad (10th generation) - 2024-05-24 at 09 16 42
Simulator Screenshot - iPad (10th generation) - 2024-05-24 at 09 16 52
Simulator Screenshot - iPhone Xs - 2024-05-24 at 09 13 58
Simulator Screenshot - iPhone Xs - 2024-05-24 at 09 14 18
Simulator Screenshot - iPhone Xs - 2024-05-24 at 09 14 41
Simulator Screenshot - iPhone Xs - 2024-05-24 at 09 15 19
Simulator Screenshot - iPhone Xs - 2024-05-24 at 09 15 38
Simulator Screenshot - iPhone Xs - 2024-05-24 at 09 16 00
Simulator Screenshot - iPhone Xs - 2024-05-24 at 09 16 30
Simulator Screenshot - iPhone Xs - 2024-05-24 at 09 16 43

**Comment by ErikEverson** [comment]
Here are screen shots of the fixes after

Simulator Screenshot - Apple TV 4K - 2024-05-24 at 08 55 20
Simulator Screenshot - Apple TV 4K - 2024-05-24 at 08 57 38
Simulator Screenshot - Apple TV 4K - 2024-05-24 at 08 58 12
Simulator Screenshot - Apple TV 4K - 2024-05-24 at 08 58 27
Simulator Screenshot - Apple TV 4K - 2024-05-24 at 08 59 09
Simulator Screenshot - Apple TV 4K - 2024-05-24 at 08 59 51
Simulator Screenshot - Apple TV 4K - 2024-05-24 at 09 00 22
Simulator Screenshot - Apple TV 4K - 2024-05-24 at 09 00 40
Simulator Screenshot - Apple TV 4K - 2024-05-24 at 09 00 57
Simulator Screenshot - Apple TV 4K - 2024-05-24 at 09 01 02
Simulator Screenshot - iPad (10th generation) - 2024-05-24 at 08 55 22
Simulator Screenshot - iPad (10th generation) - 2024-05-24 at 08 57 45
Simulator Screenshot - iPad (10th generation) - 2024-05-24 at 08 58 13
Simulator Screenshot - iPad (10th generation) - 2024-05-24 at 08 58 29
Simulator Screenshot - iPad (10th generation) - 2024-05-24 at 08 59 10
Simulator Screenshot - iPad (10th generation) - 2024-05-24 at 08 59 53
Simulator Screenshot - iPad (10th generation) - 2024-05-24 at 09 00 24
Simulator Screenshot - iPad (10th generation) - 2024-05-24 at 09 00 41
Simulator Screenshot - iPad (10th generation) - 2024-05-24 at 09 01 14
Simulator Screenshot - iPhone Xs - 2024-05-24 at 08 55 24
Simulator Screenshot - iPhone Xs - 2024-05-24 at 08 57 47
Simulator Screenshot - iPhone Xs - 2024-05-24 at 08 58 14
Simulator Screenshot - iPhone Xs - 2024-05-24 at 08 58 30
Simulator Screenshot - iPhone Xs - 2024-05-24 at 08 59 11
Simulator Screenshot - iPhone Xs - 2024-05-24 at 08 59 54
Simulator Screenshot - iPhone Xs - 2024-05-24 at 09 00 25
Simulator Screenshot - iPhone Xs - 2024-05-24 at 09 00 42

**Comment by bplattenburg** [comment]
@ErikEverson Thanks!

I'm hoping to merge the SDK version bump as its own PR in https://github.com/getditto/DittoSwiftTools/pull/138 so it gets called out in the changelog and we make sure to bump the major version here

**Comment by bplattenburg** [comment]
From our conversation, this should also fix https://github.com/getditto/DittoSwiftTools/issues/139

File: Sources/DittoPresenceViewer/PresenceView.swift [hunk 2]
```diff
@@ -19,18 +19,20 @@ import UIKit
 import AppKit
 #endif
 
-@available(macOS 10.15, *)
+#if canImport(WebKit)
+@available(iOS 13, macOS 10.15, *)
```
Comment (bplattenburg) [inline Sources/DittoPresenceViewer/PresenceView.swift:23]: I think we can drop the `iOS` piece here since we only support 14.0 and newer now

**Review by bplattenburg — CHANGES_REQUESTED** [review]

**Review by bplattenburg — APPROVED** [review]
Looks great!

File: Package.swift [hunk 3]
```diff
@@ -6,6 +6,8 @@ let package = Package(
     name: "DittoSwiftTools",
     platforms: [
         .iOS(.v14),
+        .macOS(.v11),
```
Comment (bplattenburg) [inline Package.swift:9]: Can we remove macOS support again since its still broken? We've got a long term issue tracking adding it back

**Pull request merged**